operations/companies.md

```diff
@@ -244,36 +326,196 @@ Adds a new company. Note this operation supports [Portfolio Access Tokens](../gu
 | `Department` | string | optional | The internal segmentation of a company, e.g. sales department. |
 | `DunsNumber` | string | optional | The Dun & Bradstreet unique 9-digit DUNS number. |
-| `CreditRating` | [Credit rating](#credit-rating) | optional | Credit rating to define creditworthiness of the company. |
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the company from external system. |
 | `ReferenceIdentifier` | string | optional | External system identifier - custom identifier used by an external system such as an external database. |
-| `WebsiteUrl` | string | optional | The website url of the company. |
+| `AccountingCode` | string | optional | Accounting code of the company. |
+| `AdditionalTaxIdentifier` | string | optional | Additional tax identifier of the company. |
+| `BillingCode` | string | optional | Billing code of the company. |
+| `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
+| `ContactPerson` | string | optional | Contact person of the company. |
+| `ElectronicInvoiceIdentifier` | string | optional | Electronic invoice identifier of the company. |
+| `Identifier` | string | optional | Other identifier of the company, e.g. legal identifier. |
+| `Iata` | string | optional | Iata of the company. |
+| `IsActive` | boolean | required | Whether the company is still active. |
+| `Notes` | string | optional | Additional notes. |
+| `Number` | integer | required | Unique number of the company. |
+| `TaxIdentifier` | string | optional | Tax identification number of the company. |
+| `Telephone` | string | optional | Contact telephone number. |
+| `CreatedUtc` | string | optional | Date of [Company](companies.md#company) creation date and time. |
+| `UpdatedUtc` | string | optional | Date of [Company](companies.md#company) last update date and time. |
+| `AddressId` | string | optional | Unique identifier of the company [Address](addresses.md#account-address). |
+| `MergeTargetId` | string | optional |  |
+| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of company from external system. |
+| `IsUpdatedByMe` | boolean | optional |  |
+| ~~`Address`~~ | ~~[OldAddress](configuration.md#address)~~ | ~~optional~~ | ~~Address of the customer.~~ **Deprecated!** |
+| ~~`TaxIdentificationNumber`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
 
-#### Company options parameters
+#### CompanyOptions
+Options of the company.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Invoiceable` | boolean | required | Whether the company is invoiceable or not. |
-| `AddFeesToInvoices` | boolean | required | Whether the company has an additional fee applied for invoicing or not. |
+| `Invoiceable` | boolean | required |  |
+| `AddFeesToInvoices` | boolean | required |  |
+| `AddTaxDeductedPaymentToInvoices` | boolean | required |  |
```

Comment:
❓ Added this description that I completely created based on the property name.

`"Whether tax-deducted payments should be automatically added to invoices."` 

Did I nailed it? 💅